code/modules/mob/living/simple_animal/abnormality/waw/parasite_tree.dm

```diff
@@ -0,0 +1,382 @@
+	owner.apply_status_effect(THE_TREE_CURSE)
+	qdel(src)
+
+		//CURSE EFFECT
+/datum/status_effect/parasite_tree_curse
+	id = "parasite_tree_curse"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1
+	tick_interval = 50
+	alert_type = null
+	on_remove_on_mob_delete = TRUE //Alot of people get gibbed by abnormalities, so this ensures they are removed from the blessed list.
+	var/mutable_appearance/leaf_visual
+	var/mob/living/simple_animal/hostile/abnormality/parasite_tree/connected_abno
+
+/datum/status_effect/parasite_tree_curse/on_apply()
+	. = ..()
+	leaf_visual = mutable_appearance('ModularTegustation/Teguicons/tegu_effects32x48.dmi', "hypocrisy", -HALO_LAYER)
+	leaf_visual.color = "#4B0076" //indigo
+	owner.add_overlay(leaf_visual)
+	if(locate(/mob/living/simple_animal/hostile/abnormality/parasite_tree) in GLOB.abnormality_mob_list)
+		connected_abno = locate(/mob/living/simple_animal/hostile/abnormality/parasite_tree) in GLOB.abnormality_mob_list
+		connected_abno.blessed += src
+	to_chat(owner, "<span class='warning'>You feel something sprouting under your skin. Its time to be reborn with the tree.</span>")
+
+/datum/status_effect/parasite_tree_curse/tick()
+	. = ..()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/L = owner
+		if(L.sanity_lost || L.stat == DEAD)
+			qdel(src)
+		var/tree_toxin = L.maxSanity * 0.20
+		L.apply_damage(tree_toxin, WHITE_DAMAGE, null, L.run_armor_check(null, WHITE_DAMAGE), spread_damage = FALSE)
+
+/datum/status_effect/parasite_tree_curse/on_remove()
+	owner.cut_overlay(leaf_visual)
+	var/mob/living/carbon/human/host = owner
+	if(connected_abno)
+		connected_abno.blessed -= src
+		if(!owner || host.stat == DEAD)
+			connected_abno.endBreach()
+	if(host.sanity_lost && host.stat != DEAD)
+		var/mob/living/simple_animal/hostile/parasite_tree_sapling/N = new(owner.loc)
+		nested_items(N, host.get_item_by_slot(ITEM_SLOT_SUITSTORE))
+		nested_items(N, host.get_item_by_slot(ITEM_SLOT_BELT))
+		nested_items(N, host.get_item_by_slot(ITEM_SLOT_BACK))
+		nested_items(N, host.get_item_by_slot(ITEM_SLOT_OCLOTHING))
+		QDEL_IN(owner, 5) //rabbit sanity implant explodes at 5
+	. = ..()
+
+/datum/status_effect/parasite_tree_curse/proc/nested_items(mob/living/simple_animal/hostile/nest, obj/item/nested_item)
+	if(nested_item)
+		nested_item.forceMove(nest)
+
+//Parasite Tree Ego Weapon Trap
+/obj/structure/liars_leaf
+	gender = PLURAL
+	name = "strange tree leaf"
+	desc = "A leaf from a large tree. Touching it will heal your wounds."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "liars_leaf"
+	anchored = TRUE
+	density = FALSE
+	resistance_flags = FLAMMABLE
+	max_integrity = 15
+	color = "#AAFF00"
+	var/windup = 0
+
+/obj/structure/liars_leaf/Initialize()
+	. = ..()
+	for(var/obj/structure/liars_leaf/leaf in view(1, get_turf(src)))
+		if(leaf != src)
+			qdel(src)
+	windup = world.time + (1 SECONDS)
+	QDEL_IN(src, (15 SECONDS))
+
+/obj/structure/liars_leaf/Crossed(atom/movable/AM)
```

Comment:
Might be better to transition to a "attacked_by"/"attack_hand" and check for intent?

Reply:
It is intended to be a forgiving trap. I dunno i need it to be like something people can accidentally stumble into but i want it to not be annoying. I dunno.

Reply:
Ah, cool that works for me.